bin/webpack

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env ruby
 
 ENV["RAILS_ENV"] ||= ENV["RACK_ENV"] || "development"
-ENV["NODE_ENV"] ||= ENV["RAILS_ENV"]
+ENV["NODE_ENV"]  ||= "development"
```

Comment:
Why'd you change this? Since RAILS_ENV is set above this should be fine.

Reply:
This was done automatically by the Webpacker upgrade script. It seems like it would be best to revert it as you suggest. 👍

Reply:
Ah, I did some further reading and it's intentional by the Webpacker folks since Rails can have many different custom environments but for Node, it's just dev and prod. Also, Webpack is not used anywhere but in dev anyway (all assets are compiled for staging/prod). Won't change.